test/verify/machineslib.py

```diff
@@ -1104,14 +1131,41 @@ def installWithErrorTest(dialog):
                                          start_vm=False,
                                          connection='session'))
         # try to INSTALL WITH ERROR
-        installWithErrorTest(TestMachines.VmDialog(self, "subVmTestCreate15", is_filesystem_location=True,
+        installWithErrorTest(TestMachines.VmDialog(self, "subVmTestCreate15", sourceType='file',
                                                    location=config.NOVELL_MOCKUP_ISO_PATH,
                                                    memory_size=900, memory_size_unit='GiB',
                                                    storage_size=10, storage_size_unit='MiB',
                                                    os_vendor=config.APPLE_VENDOR,
                                                    os_name=config.MACOS_X_LEOPARD,
                                                    connection='session'))
 
+        # test PXE Source
+        if self.provider == "libvirt-dbus":
+            self.machine.execute("virsh net-destroy default && virsh net-undefine default")
+
+            # Disable selinux because it makes TFTP directory inaccesible and we don't want sophisticated configuration for tests
+            self.machine.execute("setenforce 0")
```

Comment:
This needs to be guarded, as `setenforce` doesn't exist on the Debian based OSes.

Reply:
Right, thanks!